### ISSUE TITLE
New rule list-item-inner-content-indent

### ISSUE THREAD

**Issue by evilebottnawi** [wooorm/remark-lint#138]
Maybe  best name :smile: 

Correct:
```md
-   Respect that some individuals and cultures consider the casual use of profanity and sexualized language offensive
    and off-putting.
```
Incorrect:
```md
-   Respect that some individuals and cultures consider the casual use of profanity and sexualized language offensive
     and off-putting.
```

**Comment by wooorm** [comment]
Yes, wanna go for it!?

**Comment by evilebottnawi** [comment]
@wooorm yep, in near future i do this. The name we take which I suggested?

**Comment by wooorm** [comment]
:+1:

Could it be an addition to `list-item-content-indent`?

**Comment by wooorm** [comment]
Hmm, still, it’ll be better in the long run to have one rule which works as expected, instead of two half-working rules!

**Comment by evilebottnawi** [comment]
@wooorm do it this in `list-item-content-indent` :+1:

**Comment by wooorm** [comment]
@evilebottnawi Friendly ping!

**Comment by evilebottnawi** [comment]
@wooorm no time :disappointed:

**Comment by wooorm** [comment]
Closed by ab313e0, which adds the `remark-lint-no-paragraph-content-indent` rule, which works on paragraphs everywhere!

**Comment by evilebottnawi** [comment]
@wooorm can your publish this package? :smile:

**Comment by wooorm** [comment]
done! sorry 😬
